Frameworks/CoreText/CTFramesetter.mm

```diff
@@ -23,30 +23,31 @@
 
 using namespace std;
 
-@implementation _CTFrameSetter : NSObject
+@implementation _CTFramesetter : NSObject
 - (void)dealloc {
+    _typesetter = nil;
     [super dealloc];
 }
 @end
 
-static _CTFrame* _createFrame(_CTFrameSetter* frameSetter, CGRect frameSize, CGSize* sizeOut, bool createFrame) {
+static _CTFrame* _createFrame(_CTFramesetter* framesetter, CGRect frameSize, CGSize* sizeOut, bool createFrame) {
     _CTFrame* ret = nil;
 
     if (createFrame) {
         ret = [_CTFrame new];
-        ret->_frameSetter = [frameSetter retain];
```

Comment:
I changed _framesetter to a StrongId<>

---
In reply to: [79681629](https://github.com/Microsoft/WinObjC/pull/989#discussion_r79681629) [](ancestors = 79681629)